SmartTutor/ContentModel/LearningObjects/Repository/ILearningObjectRepository.cs

```diff
@@ -10,5 +10,11 @@ public interface ILearningObjectRepository
         Challenge GetChallenge(int challengeId);
         List<QuestionAnswer> GetQuestionAnswers(int questionId);
         List<ArrangeTaskContainer> GetArrangeTaskContainers(int arrangeTaskId);
+        Image GetImageForSummary(int summaryId);
+        LearningObject GetInteractiveLOForSummary(int summaryId);
+        Text GetTextForSummary(int summaryId);
+        Video GetVideoForSummary(int summaryId);
+        Challenge GetChallengeForSummary(int summaryId);
```

Comment:
There already is a GetChallenge function above, no need to add it for a summary specifically.